akka-http-core/src/main/scala/akka/http/Http.scala

```diff
@@ -33,28 +38,22 @@ class HttpExt(config: Config)(implicit system: ActorSystem) extends akka.actor.E
     val endpoint = new InetSocketAddress(interface, port)
     val effectiveSettings = ServerSettings(settings)
 
-    val connections: Source[StreamTcp.IncomingConnection, Future[StreamTcp.ServerBinding]] = StreamTcp().bind(endpoint, backlog, options, effectiveSettings.timeouts.idleTimeout)
-    val serverBlueprint: Graph[HttpServerPorts, Unit] = HttpServer.serverBlueprint(effectiveSettings, log)
+    val connections: Source[StreamTcp.IncomingConnection, Future[StreamTcp.ServerBinding]] =
+      StreamTcp().bind(endpoint, backlog, options, effectiveSettings.timeouts.idleTimeout)
+    val layer = serverLayer(effectiveSettings, log)
```

Comment:
Yes, but since #16168 has been closed yesterday I'd rather go for the proper solution now.